packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -88,8 +96,11 @@ function webpackConfigSingleSpa(opts) {
           ],
         },
       }),
-    ].concat(
-      opts.typecheck === "typescript" ? [new ForkTsCheckerWebpackPlugin()] : []
-    ),
+    ].concat(typescript ? [new ForkTsCheckerWebpackPlugin()] : []),
+    resolve: {
+      extensions: [".js", ".mjs", ".wasm", ".json"].concat(
+        typescript ? [".ts", ".tsx"] : []
```

Comment:
So that `import './component'` works for typescript projects without the file extension